novawallet/Modules/Notifications/StakingRewardsNotifications/StakingRewardsNotificationsInteractor.swift

```diff
@@ -13,8 +13,7 @@ final class StakingRewardsNotificationsInteractor {
     private func subscribeChains() {
         chainRegistry.chainsSubscribe(
             self,
-            runningInQueue: .main,
-            filterStrategy: .enabledChains
```

Comment:
I think we need also remove filters for governance push notifications

Reply:
we don't use filters for governance notifications